docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -6139,29 +6033,29 @@
        "                <table style=\"width: 100%; text-align: left;\">\n",
        "                    <tr>\n",
        "                        <td style=\"text-align: left;\">\n",
-       "                            <strong>Comm: </strong> tcp://127.0.0.1:33491\n",
+       "                            <strong>Comm: </strong> tcp://127.0.0.1:45093\n",
        "                        </td>\n",
        "                        <td style=\"text-align: left;\">\n",
        "                            <strong>Total threads: </strong> 1\n",
        "                        </td>\n",
        "                    </tr>\n",
        "                    <tr>\n",
        "                        <td style=\"text-align: left;\">\n",
-       "                            <strong>Dashboard: </strong> <a href=\"http://127.0.0.1:34333/status\" target=\"_blank\">http://127.0.0.1:34333/status</a>\n",
+       "                            <strong>Dashboard: </strong> <a href=\"http://127.0.0.1:44017/status\" target=\"_blank\">http://127.0.0.1:44017/status</a>\n",
        "                        </td>\n",
        "                        <td style=\"text-align: left;\">\n",
-       "                            <strong>Memory: </strong> 62.82 GiB\n",
+       "                            <strong>Memory: </strong> 125.52 GiB\n",
        "                        </td>\n",
        "                    </tr>\n",
        "                    <tr>\n",
        "                        <td style=\"text-align: left;\">\n",
-       "                            <strong>Nanny: </strong> tcp://127.0.0.1:43093\n",
+       "                            <strong>Nanny: </strong> tcp://127.0.0.1:41841\n",
        "                        </td>\n",
        "                        <td style=\"text-align: left;\"></td>\n",
        "                    </tr>\n",
        "                    <tr>\n",
        "                        <td colspan=\"2\" style=\"text-align: left;\">\n",
-       "                            <strong>Local directory: </strong> /home/mmccarty/sandbox/rapids/cudf/docs/cudf/source/user_guide/dask-worker-space/worker-jsuvfju4\n",
+       "                            <strong>Local directory: </strong> /home/bdice/rapids1/cudf/docs/cudf/source/user_guide/dask-worker-space/worker-8ci7x8v3\n",
```

Comment:
@shwina FYI this is the only user path that I see left in the notebook.

Reply:
It looks like this doesn't end up in the rendered notebook somehow?

Reply:
It's hidden in some interactive HTML.

![image](https://user-images.githubusercontent.com/3943761/165362162-45ecbe03-d8f8-4a87-8415-4fa037f985fd.png)

Reply:
Ah I see it -- let's get rid of it then, or comment it out?

Reply:
Fixed in 59bdba5abfe9adaef86b0a354572f6cd7e14e110.